### ISSUE TITLE
lunotriquetral ligament tear

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10726]
**Preferred term label:**
lunotriquetral ligament tear

**Synonyms**

**Definition (free text, please give PubMed ID)**
Advanced imaging revealing partial or complete LT ligament tear

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Lunotriquetral (LT) ligament injury

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001207
- label: Lunotriquetral ligament tear
- parents: [Abnormal carpal morphology (HP:0001191)](https://hpo.jax.org/app/browse/term/HP:0001191)
- pmid: PMID:36689137
- def: A breakage (partial or complete tear) of the lunotriquetral ligament. The most common cause is physical trauma to the wrist such as a fall onto an outstretched hand. Rupture or tear of the the lunotriquetral ligament may lead to carpal instability.
- comment: The scapholunate and lunotriquetral interosseous ligaments form the interconnections between the bones of the proximal carpal row and share several anatomic features. Each forms a barrier between the radiocarpal and midcarpal joints, connecting the dorsal, proximal, and palmar edges of the respective joint surfaces.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10726
- synonyms:
